.github/workflows/translations.yml

```diff
@@ -0,0 +1,24 @@
+name: Translations
+
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout the source code
+      uses: actions/checkout@master
+    - name: Install Python packages
+      run: |
+        pip3 install wheel
+        pip3 install setuptools
+        pip3 install pygithub
+        pip3 install pygithub3
+    - name: Deploy
+      if: github.repository == 'zen-mod/ZEN' && ! contains(github.event.head_commit.message, '[ci skip]')
```

Comment:
Is `ci skip` even necessary at any point? For some other builds ok, but for this? And as it's running only on master and master is only updated via PRs, then added with squash merging, the PR should start with `ci skip`.

Reply:
No, this is used in ACE3 for documentation updates via CI. There is no reason to not include it here though, currently Actions do not support `ci skip` natively.

Reply:
CI skip just doesn't go into this project and can be safely removed and by any chance, a different doc system will be chosen, it will damn sure not push commits to master.

Reply:
There is no harm in leaving the "ci skip" check in.